pkg/console/operator/sync_v400.go

```diff
@@ -408,3 +410,31 @@ func getDeploymentGeneration(co *consoleOperator) int64 {
 	}
 	return deployment.Generation
 }
+
+func copyCustomLogoConfigMap(co *consoleOperator, operatorConfig *operatorv1.Console) {
+	// Check if a custom logo is set on the operator config
+	logoName := operatorConfig.Spec.Customization.CustomLogoFile.Name
```

Comment:
This should return an `error` like other funcs that make API calls so it can bubble up.